pkgs/top-level/all-packages.nix

```diff
@@ -12515,7 +12515,7 @@ in
 
   hercules-ci-agent = callPackage ../development/tools/continuous-integration/hercules-ci-agent { };
 
-  niv = haskell.lib.justStaticExecutables haskellPackages.niv;
+  niv = (haskell.lib.justStaticExecutables haskellPackages.niv).bin;
```

Comment:
Shoudln't we prefer `lib.getBin` here as that will then survive the next refactoring of the haskell package set?

Reply:
Good idea. Updated.